### ISSUE TITLE
About Half the Examples I tried crashed at bulletjme.dll+0x79570

### ISSUE THREAD

**Issue by tdhargraves** [stephengold/Minie#23]
I just downloaded Minie to a Windows 10 system.
About 50% of the examples failed with the same error, at bulletjme.dll+0x79570:

```console
# A fatal error has been detected by the Java Runtime Environment:
#
#  EXCEPTION_ACCESS_VIOLATION (0xc0000005) at pc=0x00007ffea6b09570, pid=11868, tid=13056
#
# JRE version: OpenJDK Runtime Environment (11.0.6+10) (build 11.0.6+10)
# Java VM: OpenJDK 64-Bit Server VM (11.0.6+10, mixed mode, tiered, compressed oops, g1 gc, windows-amd64)
# Problematic frame:
# C  [bulletjme.dll+0x79570]
```

For example:
TestSoftBody crashed at startup.
ConveyorDemo crashed when the block landed on the conveyor.
TestSoftBodyControl worked fine.

I repeated these test runs using a clean build, using the supplied MinieExamples.bat script in distributions.
I got the same crashes using Java runtime 1.8.9_301, SDK 8.0_301, SDK 11.0.6
This is the output from ConveyorDemo:

```console
C:\Users\tom_h\Desktop\MinieExamples\bin>MinieExamples.bat
Libbulletjme version 14.1.0 initializing
Mar 28, 2022 2:08:21 PM com.jme3.input.InputManager addMapping
WARNING: Attempted to add mapping "ScreenShot" twice to trigger.
C:\Users\tom_h\Desktop\MinieExamples\bin>MinieExamples.bat
Libbulletjme version 14.1.0 initializing
Mar 28, 2022 2:09:36 PM com.jme3.input.InputManager addMapping
WARNING: Attempted to add mapping "ScreenShot" twice to trigger.
Mar 28, 2022 2:09:36 PM jme3utilities.SignalTracker test
WARNING: Testing a signal which has not yet been added: "orbitLeft".
Mar 28, 2022 2:09:36 PM jme3utilities.SignalTracker test
WARNING: Testing a signal which has not yet been added: "orbitRight".
[ALL OK UNTIL I PRESS SPACE BAR, THE BLOCK DROPPED AND THE APP CRASHED]
#
# A fatal error has been detected by the Java Runtime Environment:
#
#  EXCEPTION_ACCESS_VIOLATION (0xc0000005) at pc=0x00007ffea1d89570, pid=11120, tid=0x000000000000217c
#
# JRE version: Java(TM) SE Runtime Environment (8.0_301-b09) (build 1.8.0_301-b09)
# Java VM: Java HotSpot(TM) 64-Bit Server VM (25.301-b09 mixed mode windows-amd64 compressed oops)
# Problematic frame:
# C  [bulletjme.dll+0x79570]
#
# Failed to write core dump. Minidumps are not enabled by default on client versions of Windows
#
# An error report file with more information is saved as:
# C:\Users\tom_h\Desktop\MinieExamples\bin\hs_err_pid11120.log
#
# If you would like to submit a bug report, please visit:
#   http://bugreport.java.com/bugreport/crash.jsp
# The crash happened outside the Java Virtual Machine in native code.
# See problematic frame for where to report the bug.
#
```

**Comment by stephengold** [comment]
Thanks for documenting this issue. I'll try to reproduce it ASAP.

**Comment by stephengold** [comment]
I reproduced the crash on my Windows 11 laptop. I suspect it's caused by casting pointers to `long` instead of `jlong` in the native library. (On 64-bit Windows systems, `long` is only 32 bits.)

I'll build corrected native libraries and see if they resolve the issue for me.

**Comment by stephengold** [comment]
Note to self: compare with https://github.com/stephengold/Minie/issues/2

**Comment by stephengold** [comment]
committed a fix at a510e937d3f2e34a23ac8a61d12fb405deee01aa
working toward a v4.8.1 patch release

Since there seems to be a recurring issue with casting pointers to `long`, I added a continuous-integration check to Libbulletjme that fails if the word `long` appears in the native source.

**Comment by stephengold** [comment]
@tdhargraves please confirm this issue is fixed in Minie v4.8.1
